CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+cmake_minimum_required(VERSION 3.8)
+project(cuda-fixnum CXX CUDA)
+
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CUDA_STANDARD 11)
+set(EXECUTABLE_OUTPUT_PATH "${PROJECT_BINARY_DIR}/bin")
+
+# CUDA
+find_package(CUDA REQUIRED)
```

Comment:
According to https://cmake.org/cmake/help/v3.15/module/FindCUDA.html, by listing `CUDA` up there in `project`, this `find_package` should be unnecessary. Maybe this is a source of some of cmake's confusion?